packages/Angular/Explorer/base-forms/src/lib/base-form-component.ts

```diff
@@ -180,10 +180,19 @@ export abstract class BaseFormComponent extends BaseRecordComponent implements A
     return this._isFavorite;
   }
 
+  private _userPermissions: {permission: EntityPermissionType, canDo: boolean}[] = [];
```

Comment:
Alternatively, could use a map since we only need one bool per unique `EntityPermissionType`

```ts
  private _userPermissions: Record<EntityPermissionType, boolean> = {};
```

More in the next comment...